fabric-datafixer-v1/src/main/java/net/fabricmc/fabric/api/datafixer/v1/SimpleFixes.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.datafixer.v1;
+
+import com.google.common.collect.ImmutableMap;
+import com.mojang.datafixers.DataFixerBuilder;
+import com.mojang.datafixers.Dynamic;
+import com.mojang.datafixers.schemas.Schema;
+import com.mojang.datafixers.util.Pair;
+
+import net.fabricmc.fabric.impl.datafixer.FabricSimpleFixes;
+import net.minecraft.nbt.Tag;
+
+/**
+ * This class contains several common datafixes modders would use.
+ */
+public interface SimpleFixes {
+
+	public static final SimpleFixes INSTANCE = FabricSimpleFixes.INSTANCE;
+
+	/**
+	 * A basic DataFix for changing block names.
+	 * 
+	 * @param builder The builder to add this fix to.
+	 * @param name The name of the datafix (this has no effect on actual process)
+	 * @param changes A map of all changed values, where the key should be original and value is new value.
+	 * @param schema The Schema to apply this fix to.
+	 */
+	public abstract void addBlockRenameFix(DataFixerBuilder builder_1, String name, String oldId, String newId, Schema schema_1);
```

Comment:
The javadoc is missing oldId and newId.

Reply:
Consider fixing the `_#` suffixes of parameters.